build/signedbuild.yml

```diff
@@ -154,9 +146,9 @@ jobs:
       uploadAPIScan: false
       uploadBinSkim: false
       uploadFortifySCA: false
+      uploadFxCop: false
       uploadModernCop: false
       uploadPREfast: false
-      uploadRoslyn: false
```

Comment:
Is it true scans that aren't listed are uploaded?

Reply:
Yes.